### DISCUSSION TITLE
Design spec for text input fields

### DISCUSSION THREAD

**Discussion by mairin** [podman-desktop/podman-desktop#3235] [Design Feedback]
Hi everyone :wave:

I just put together a little spec to help make Podman Desktop's text input components more consistent. Let me know what you think. If you have any issues with visibility / contrast / accessibility with our text inputs today, I'd especially love to hear your thoughts.

The spec is visible here:
https://design.penpot.app/#/view/8ec95363-4e2c-80a4-8002-cb67c874d0be?page-id=42e69059-b409-807e-8002-d0b98cda8446&section=interactions&index=0&share-id=c65f3168-697c-808c-8002-d0c926187cdf

The ticket is here:
https://github.com/containers/podman-desktop/issues/3234

Here's an image preview of the spec:
![Text Input (1)](https://github.com/containers/podman-desktop/assets/799683/3d6feaba-e550-4c45-8db2-b1d95ae1f448)

**Comment by rujutashinde** [comment]
closing this discussion as it is either implemented or not needed
